android/Staccato_AN/app/src/main/java/com/on/staccato/presentation/timeline/TimelineFragment.kt

```diff
@@ -82,4 +88,25 @@ class TimelineFragment :
             }
         }
     }
+
+    private fun View.inflateCreationMenu(): PopupMenu {
+        val popup =
+            PopupMenu(
+                this.context,
+                this,
+                Gravity.END,
+                0,
+                R.style.Theme_Staccato_AN_PopupMenu,
+            )
+        popup.menuInflater.inflate(R.menu.menu_sort, popup.menu)
+        return popup
+    }
```

Comment:
```suggestion
    private fun View.inflateCreationMenu(): PopupMenu =
        PopupMenu(
            this.context,
            this,
            Gravity.END,
            0,
            R.style.Theme_Staccato_AN_PopupMenu,
        ).apply {
            menuInflater.inflate(R.menu.menu_sort, menu)
        }
```
apply를 사용해보는 건 어떤가요!

Reply:
앗..! 🤣 저는 scope function보다는 변수 이름을 명확하게 지어서 사용하는 걸 선호합니다!
scope function을 잘못 쓰면 가독성이 떨어진다고 생각해서 scope function 사용은 꼭 필요한 상황이 아니면 지양하는 편입니다..!

Reply:
그렇다면 기존 방식대로 가죠!